packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -12,7 +12,7 @@ import { Classes as CoreClasses, IProps, Popover, Position } from "@blueprintjs/
 
 import * as Classes from "../common/classes";
 import { LoadableContent } from "../common/loadableContent";
-import { HeaderCell, IHeaderCellProps } from "./headerCell";
+import { HeaderCell, IHeaderCellProps, IInternalHeaderCellProps } from "./headerCell";
```

Comment:
It's kind of both.

`IHeaderCellProps` is exposed by the generic `headerCell.tsx` for use by two different but very similar components, `RowHeaderCell` and `ColumnHeaderCell`, as a means to expose both of which are used by consumers.

`IInternalHeaderCellProps` is exposed by the generic `headerCell.tsx` to allow `RowHeaderCell` and `ColumnHeaderCell` to internally provide more props to `HeaderCell` than the ones surfaced to consumers.

The first one is kind of `Core`-ish, and the second one seems definitely `Internal`. Thoughts?